CodeYourFuture/cyf-forms
# Chore/fix deps

This is a:






### Description



- **Purpose** - 

CircleCI is configured to use node v11.12.
The build now fails as eslint-plugin-jsx-a11y v6.6.0 relies on axe-core ^4.4.2 and axe-core requires node ^12.
This chore is to temporarily fix versions to resolve this until the node version is upgraded.

- **How to check** - 

- Switch to qa branch
- Set your node version to 11.12
- Delete node_modules and yarn.lock
- run `yarn`
It should fail.

- Switch to this branch
- run `yarn`, then `yarn build:qa` 
Both should complete successfully.

### Links



### Author checklist



- [x] I have made the PR to `staging` from a branch named `<category>/<name>`, e.g. `feature/edit-spaceships` or `bugfix/restore-oxygen`
- [ ] I have completed the manual tests [described here](https://github.com/CodeYourFuture/tech-team/wiki/Manual-Test-Procedures)


- [ ] I have updated the Swagger API documentation in `src/swagger/routes/`
- [ ] I have explicitly tested the access rights for _at least_ unauthenticated, admin and non-admin cases

### Reviewer checklist
Remember that asking questions helps everyone understand why decisions have been made, so the next person can see too!
- **Pull request** - read the content of the PR itself (_Conversation_ tab) and consider the following questions:
    - Is the user story/ticket linked?
    - Does the description match the linked item?
    - Does the description tell you how to validate the work?
    - Does the build still work (_Checks_ tab)?
    - What approach would you take to meet this goal?
- **Code changes** - read through the changeset (_Files changed_ tab) and consider the following questions:
    - Do the code changes match the description?
    - Are there any changes that should not be part of this pull request (e.g. layout changes in files not related to the functionality)?
    - Do you understand what the changes are doing, how they meet the goal of the PR?
    - Does the approach to meeting those goals seem sensible? If it's not the one you thought of, is it significantly better or worse?
    - Is this well-structured code (consistent layout, comprehensible variable names, relatively small files)?
- **Software** - check out the PR branch, run the software and consider the following questions:
    - Does the product still start and run correctly?
    - Is the goal of the pull request met (i.e. new behaviour for a new feature, changed behaviour for a bug fix, identical behaviour for a refactor)?
    - In the parts of the product this PR touched:
        - Is the spelling, punctuation and grammar for user-facing text correct?
        - Does the layout/UI match the designs?
- **Project progress**
    - Have new tickets been created to cover any additional actions that have appeared during this review?
